scripts/start_docker_bblfsh.sh

```diff
@@ -8,19 +8,15 @@ image="bblfsh/bblfshd:v2.4.2"
 
 set -ev
 
-if [[ -z "${STYLE_CHECK}" ]]; then
-  echo "Starting Docker image ${image}"
-  mkdir -p $HOME/bblfsh-drivers
-  docker run -v $HOME/bblfsh-drivers:/var/lib/bblfshd --name bblfshd --privileged -p 9432:9432 -d "${image}"
+echo "Starting Docker image ${image}"
+mkdir -p $HOME/bblfsh-drivers
+docker run -v $HOME/bblfsh-drivers:/var/lib/bblfshd --name bblfshd --privileged -p 9432:9432 -d "${image}"
 
-  if [[ "$?" -ne 0 ]]; then
-    echo "Unable to start Docker ${file_list}" >&2
-    exit "${E_BAD_DOCKER}"
-  fi
-
-  docker exec -it bblfshd bblfshctl driver install --recommended
-else
-  echo "Skip starting Docker image ${image}"
+if [[ "$?" -ne 0 ]]; then
+  echo "Unable to start Docker ${file_list}" >&2
+  exit "${E_BAD_DOCKER}"
 fi
 
+docker exec -it bblfshd bblfshctl driver install --recommended
+
```

Comment:
While we are still at it, to make it bullet-poof, why do not we add here [return value check](https://google.github.io/styleguide/shell.xml?showone=Checking_Return_Values#Checking_Return_Values), same as for `docker run` above?